cmd/in/main.go

```diff
@@ -4,11 +4,13 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"log"
 	"os"
 
-	"github.com/starkandwayne/bosh2-errand-resource/bosh"
-	"github.com/starkandwayne/bosh2-errand-resource/concourse"
-	"github.com/starkandwayne/bosh2-errand-resource/in"
+	"github.com/cloudfoundry-community/bosh2-errand-resource/bosh"
+	"github.com/cloudfoundry-community/bosh2-errand-resource/concourse"
+	"github.com/cloudfoundry-community/bosh2-errand-resource/in"
+	proxy "github.com/cloudfoundry/socks5-proxy"
```

Comment:
I'm still learning how go imports things.  Can you explain to me the syntax here for why we need the word proxy before the address to the github.com/cloudfoundry/socks5-proxy resource?

Reply:
This allows you to alias the import. By default you’d reference this library by `socks5` I think.

Reply:
👍